### PR TITLE
chore(litmus-portal): Issue #2103 added validation for Username

### PR DESCRIPTION
**Pull request by jabhishek87**
## Proposed changes

Improved the username input field of Create New User
section(Settings Page)
close issue #2103 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ X] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ X] I have signed the commit for DCO to be passed.
- [ X] Lint and unit tests pass locally with my changes
- [ X] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ X] I have added necessary documentation (if appropriate)

## Special notes for your reviewer:
First-time contribuor

### CODE REVIEW

**Comment by jabhishek87** [comment]
can somebody review @Saranya-jena

File: litmus-portal/frontend/src/utils/validate.ts [hunk 1]
```diff
@@ -19,6 +19,15 @@ export const validateEmail = (value: string) => {
   return false;
 };
 
+export const validateUsername = (value: string) => {
+  const usernameValid = /^[a-zA-Z0-9]+$/;
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/utils/validate.ts:23]: Username regex validation would require more checks like having special characters (Not all), - and @ are an important one.

**Review by S-ayanide — CHANGES_REQUESTED** [review]
Thanks for sending a PR 🎉 

We'd love to add a new validation to the list. Do consider checking the requests our maintainers make and keep the PR fresh in order to get this merged as soon as possible.

File: litmus-portal/frontend/src/utils/validate.ts (hunk 1 above)
Comment (jabhishek87) [reply litmus-portal/frontend/src/utils/validate.ts:23]: Hi @S-ayanide It is not emailed it is a username ,  as per the Issue The allowed characters are Alphabets, and Numeric

**Review by S-ayanide — DISMISSED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Review by Saranya-jena — COMMENTED** [review]
Can you update your branch and resolve the conflicts?

**Comment by jabhishek87** [comment]
> Can you fix the deepScan issue and resolve the conflicts?

fixed

**Comment by jabhishek87** [comment]
i messed up will create new PR

**Comment by jabhishek87** [comment]
opened another pull request https://github.com/litmuschaos/litmus/pull/2217

**Pull request closed without merging**